src/main/java/cinema/Main.java

```diff
@@ -42,12 +43,13 @@ public static void main(String[] args) {
 
         AuthenticationService service = (AuthenticationService) injector
                 .getInstance(AuthenticationService.class);
-        service.register("Human", "qwerty");
-        try {
-            System.out.println(service.login("Human", "qwerty"));
-            System.out.println(service.login("Human", "qwerty1"));
```

Comment:
did you stop testing your login? you'r gonna merge this to master

Reply:
moved it back